python/ml4ir/base/model/optimizers/optimizer.py

```diff
@@ -2,6 +2,19 @@
 from tensorflow.keras.optimizers.schedules import ExponentialDecay
 from ml4ir.base.model.optimizers import cyclic_learning_rate
 from ml4ir.base.config.keys import OptimizerKey, LearningRateScheduleKey, CyclicLearningRateType
+import tensorflow as tf
+
+class DefaultValues(object):
+  """Default values for unspecified parameters of the optimizer and learning rate schedule."""
+  GRADIENT_CLIP_VALUE = 5.0
+  CONSTANT_LR = 0.01
+  EXP_DECAY_STEPS = 100000
+  EXP_DECAY_RATE = 0.96
+  CYCLIC_INITIAL_LEARNING_RATE = 0.001
+  CYCLIC_MAXIMAL_LEARNING_RATE = 0.01
```

Comment:
why do we need default values for all these? I appreciate the work, but why not simply fail the code when they are not specified? We should not be doing this under-the-hood. There should be a minimal amount of values that the user specifies. IMO, `initial` and `maximal_lr` should belong in this family.

Reply:
I think these are the tensorflow defaults @balikasg